packages/renderer/src/lib/feedback/GitHubIssueFeedback.svelte

```diff
@@ -0,0 +1,91 @@
+<script lang="ts">
+import { Button, Dropdown, ErrorMessage, Link } from '@podman-desktop/ui-svelte';
+
+import type { FeedbackCategory } from '/@api/feedback';
+
+import Dialog from '../dialogs/Dialog.svelte';
+
+export let displayModal = false;
+
+const DEFAULT_CATEGORY: FeedbackCategory = 'developers';
+export let category: FeedbackCategory = DEFAULT_CATEGORY;
+
+const FEEDBACK_CATEGORIES = new Map<FeedbackCategory, string>([
+  ['developers', '💬 Direct your words to the developers'],
+  ['feature', '🚀 Feature request'],
+  ['bug', '🪲 Bug'],
+]);
+
+let issueTitle = '';
+let issueDescription = '';
+let issueValidaionError = '';
+
+$: if (!issueTitle) {
+  if (!issueDescription) {
+    issueValidaionError = 'Please enter bug title and description';
+  } else {
+    issueValidaionError = 'Please enter bug title';
+  }
+} else {
+  if (!issueDescription) {
+    issueValidaionError = 'Please enter bug description';
+  }
+}
+
+function hideModal(): void {
+  displayModal = false;
+
+  // reset fields
+  category = DEFAULT_CATEGORY;
+  issueTitle = '';
+  issueDescription = '';
+}
+
+async function openGitHubIssues(): Promise<void> {
+  displayModal = false;
+  await window.openExternal('https://github.com/containers/podman-desktop/issues');
+  displayModal = true;
+}
+
+async function sendToGitHub(): Promise<void> {
+  displayModal = false;
+  let linkTitle = issueTitle.split(' ').join('+');
+  let linkDescription = issueDescription.split(' ').join('+');
+  const bugLink = `https://github.com/containers/podman-desktop/issues/new?template=bug_report.yml&title=${linkTitle}&bug-description=${linkDescription}`;
+  await window.openExternal(bugLink);
+  displayModal = true;
+}
+</script>
+
+<Dialog title="Share your feedback" on:close={() => hideModal()}>
+  <svelte:fragment slot="content">
+    <label for="category" class="block mb-2 text-sm font-medium text-[var(--pd-modal-text)]">Category</label>
+    <Dropdown name="category" bind:value={category}
+    options={Array.from(FEEDBACK_CATEGORIES).map(e => ({ value: e[0], label: e[1] }))}>
+    </Dropdown>
```

Comment:
issue: see https://github.com/podman-desktop/podman-desktop/pull/9833/files#r1838514274, we might want to avoid duplicating the dropdown, and have it inside the parent component